bears/php/PHPMessDetectorBear.py

```diff
@@ -0,0 +1,38 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import \
+    DistributionRequirement
+
+
+@linter(executable='phpmd',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+)(?P<message>.*)')
```

Comment:
``` yash  (e) venv  ~  coala -V -b PHPMessDetectorBear -f s.php --flush-cache
[WARNING][21:11:32] Implicit 'Default' section inheritance is deprecated. It will be removed soon. To silence this warning remove settings in the 'Default' section from your coafile. You can use dots to specify inheritance: the section 'all.python' will inherit all settings from 'all'.
Please enter a value for the setting "phpmd_ruleset" (A ruleset filename or a comma-separated string of rulesetfilenames. Available rulesets: cleancode, codesize, controversial, design, naming, unusedcode.) needed by PHPMessDetectorBear for section "cli": 
cleancode
[DEBUG][21:11:37] Platform Linux -- Python 3.5.2, coalib 0.11.0.dev99999999999999
[DEBUG][21:11:37] The file cache was successfully flushed.
Executing section cli...
[DEBUG][21:11:37] Files that will be checked:
/home/yash/s.php
[DEBUG][21:11:37] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][21:11:37] Running bear PHPMessDetectorBear...
[DEBUG][21:11:37] Running 'phpmd /home/yash/s.php text cleancode'

s.php
|   8| ········}·else·{
|    | [NORMAL] PHPMessDetectorBear:
|    | The method bar uses an else expression. Else is never necessary and you can simplify the code to work without else.
|    | *0: Do nothing
|    |  1: Open file(s)
|    |  2: Add ignore comment
|    | Enter number (Ctrl-D to exit): 0```